docs/enterprise-v2.7.md

```diff
@@ -34,6 +34,33 @@ Key highlights of this release:
 3. For Kubernetes users, please follow [the guide](/k8s-helm#v5-to-v6) carefully since a few timeplusd built-in users are removed in the new helm chart, and you can configure ingress for Appserver and Timeplusd independently.
 
 ## Releases
+### 2.7.6 {#2_7_6}
+Released on 04-30-2025. Installation options:
+* For Linux or Mac users: `curl https://install.timeplus.com/2.7 | sh` [Downloads](/release-downloads#2_7_6)
+* For Kubernetes users: `helm install timeplus/timeplus-enterprise --version v6.0.11 ..`
+* For Docker users (not recommended for production): `docker run -p 8000:8000 docker.timeplus.com/timeplus/timeplus-enterprise:2.7.6`
+
+Component versions:
+* timeplusd 2.7.45
+* timeplus_web 2.2.12
+* timeplus_appserver 2.2.13
+* timeplus_connector 2.2.8
+* timeplus cli 1.2.12
+
+#### Changelog {#changelog_2_7_6}
+
+Compared to the [2.7.5](#2_7_5) release:
+* timeplusd 2.7.37 -> 2.7.45
+  * fix global aggregation with `EMIT ON UPDATE` in multi-shard environments
+  * fix concurrency issues in hybrid aggregation
+  * support incremental checkpoints for hybrid hash join
+  * fix grouping key issues in hybrid aggregation
+  * support setting of mv_preferred_exec_node in materialized view
```

Comment:
After this PR is merged, I will add new docs about `mv_preferred_exec_node`. Otherwise this change log is too abstract